.github/workflows/twoops-tracker-cd.yml

```diff
@@ -0,0 +1,61 @@
+name: TwoopsTracker | Continuous Deployment
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'twoops_tracker/py/VERSION'
+env:
+  DOKKU_REMOTE_BRANCH: "master"
+  DOKKU_REMOTE_URL: "ssh://dokku@dokku-1.dev.codeforafrica.org/twoopstracker"
+  GIT_PUSH_FLAGS: "--force"
+  IMAGE_NAME: "codeforafrica/twoopstracker"
+  VERSION: cat twoops_tracker/py/VERSION
+
+jobs:
+  build:
+    name: Build and Deploy
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.10]
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.10.0
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Lint
+        run: |
+          ./pants lint twoops_tracker/py::
+
+      - name: Format
+        run: |
+          ./pants fmt twoops_tracker/py::
```

Comment:
1. Do we want to `fmt` in CI? i.e. if the code is formatted, aren't those changes lost since we don't commit? I think `fmt` should be run as git hook on every commit. Unless, there is a way of running it in check only.
2. We should run [`lint` in check only mode](https://www.pantsbuild.org/docs/reference-lint#skip_formatters).

Reply:
>We should run [lint in check only mode](https://www.pantsbuild.org/docs/reference-lint#skip_formatters).

seems like that's the default. No?

Reply:
As long as there is no `fmt` step, it could be.

Reply:
also, the `skip_formatters` option is not available for pants `2.10.0` which we are currently using. I'll add it on a different PR (updating packages)